api/app.py

```diff
@@ -20,21 +26,51 @@ def analyze():
     go_list1 = data["sample1"]
     go_list2 = data["sample2"]
 
-    not_present = find_non_existing_terms(go_list1, GO_DAG)
-    not_present.update(find_non_existing_terms(go_list2, GO_DAG))
+    id = str(uuid.uuid4())
 
-    result = run_comparison(go_list1, go_list2, GO_DAG)
+    thread = Compute(go_list1, go_list2, id)
+    thread.start()
 
     return {
-        "similarity": {
-            "biological_process": result[0],
-            "cellular_component": result[1],
-            "molecular_function": result[2]
-        },
-        "invalid": list(not_present)
+        "analysis_id": id
     }
 
 
+@app.route('/progress/<id>', methods=["POST"])
+@cross_origin()
+def progress(id):
+    print(RESULTS)
```

Comment:
is this print statement a debugging artifact?

Reply:
This is indeed a debugging artificat :) I removed it